benchmarks/accept_scaling.py

```
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+"""Benchmark: accept-path scaling with revision count (ISSUES.md #57).
+
+Builds a large multi-run document (via ``batch_edit_scaling.build_large_doc``),
+lays down an N-operation ``batch_edit`` redline (one changeset, N groups, 2N
+revisions), then times three ways to resolve it on identical revisions:
+
+    * ``accept_all()``        — the multi-pass baseline (never regressed)
+    * ``accept_changeset()``  — resolves the whole batch's changeset
+    * ``accept_group()`` ×N   — group-by-group resolution loop
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)